### DISCUSSION TITLE
Allow filtering by model file size

### DISCUSSION THREAD

**Discussion by nabileisa** [civitai/civitai#167] [Ideas]
Hi, Some people may have cards with 6gb or less of VRAM and can not load models above certain sizes , it would be helpful if there exists a slider to filter the models by the model current version file size, 
An additional idea would be to also allow filtering by if a `Pruned` version exists or not of the model

that would be very helpful for users in finding models that match their devices capabilities
Thanks

**Comment by JustMaier** [comment]
What if we gave you a way to request a prune? I'd have to look into it more, but I'd imagine our model scanner could also handle pruning models on-demand.

So basically, you'd make your request, wait an hour or so, then it'd be available if the prune was successful.

**Reply by nabileisa** [reply]
That would actually be a great feature and would negate the need for my initial request since all models that can be pruned will be pruned at one point.
Better yet, Model authors could use the same mechanism to generate pruned versions of their models and not have to upload both versions if they wish.
I Look forward to it.
